### PR TITLE
unified cgroup support

### LINKED ISSUE #3750: Netdata's cgroup tracking does not work with version 2 cgroups.
cgroup v2 setups preferentially use a unified hierarchy (which Netdata can easily be adapted to by just pointing all the controllers at the root), and additionally changed the names and contents of some of the files:

* The `blkio` controller is called `io` in cgroup v2, and now uses a single `io.stat` file instead of the various `blkio.*.io_service*` files.
* The `cpu` and `cpuacct` controllers got unified into one controller in cgroup v2, and now has a single file `cpu.stat` that contains both the fields from the old `cpu.stat` file, as well as microsecond resolution usage counters for total, system, and user time.
* The `memory` controller has changed enough about the data format that it needs a different parser (but now has a lot more info that can be parsed, including info on the types of allocations for the cgroup, and swap usage.
* The `devices` controller is gone (because there are better ways no to handle what it was trying to do).

### PR DESCRIPTION
**Pull request by skrzyp1**
##### Summary
Fixes #3750

### CODE REVIEW

**Comment by cakrit** [comment]
I edited the first comment to add the reference to the issue.

**Comment by skrzyp1** [comment]
@vlvkobal I can implement them too, just wanted to have this part merged so there will be no merge conflicts when other people edit. I can do that as a separate issue is that ok?

File: collectors/cgroups.plugin/sys_fs_cgroup.c [hunk 1]
```diff
@@ -528,6 +579,42 @@ static inline void cgroup_read_cpuacct_stat(struct cpuacct_stat *cp) {
...
+        }
+
+        ff = procfile_readall(ff);
+        if(unlikely(!ff)) {
+            cp->updated = 0;
+            cgroups_check = 1;
+            return;
+        }
+
+        unsigned long i, lines = procfile_lines(ff);
```
Comment (cakrit) [inline collectors/cgroups.plugin/sys_fs_cgroup.c]: Please fix warning
```
/home/christopher/CLionProjects/netdata/collectors/cgroups.plugin/sys_fs_cgroup.c: In function ‘cgroup2_read_cpuacct_stat’:
/home/christopher/CLionProjects/netdata/collectors/cgroups.plugin/sys_fs_cgroup.c:600:23: warning: unused variable ‘i’ [-Wunused-variable]
         unsigned long i, lines = procfile_lines(ff);
```

**Comment by vlvkobal** [comment]
> I can do that as a separate issue is that ok?
It is OK. But please update the documentation in this PR.

**Comment by skrzyp1** [comment]
@cakrit I removed unused variable. @vlvkobal I added info to docs. Is everything ok now?

**Pull request merged**